Application/Source/Detail/View/DetailViewModel.swift

```diff
@@ -25,6 +25,21 @@ class DetailViewModel: ViewModel, SelectionPresentingViewModel {
         return .empty()
     }
 
+    let content: BehaviorRelay<[Food]> = BehaviorRelay(value: [.beans, .greens, .potatoes, .tomatoes])
+
+    let contentListSeparator: String = ", "
+
+    private(set) lazy var contentsListText: Property<String> = {
+        let initial = ""
+        let observable = content.map { [weak self] contents -> String in
+            guard let self = self else { return initial }
+            return contents
+                .map { $0.name.value }
+                .joined(separator: self.contentListSeparator)
+        }
+        return Property(observable, initial: initial)
```

Comment:
If we remove the guard statement above (after removing the `contentListSeparator` property) we could just define the `initial` string in line here.